readme.md

```diff
@@ -10,135 +10,83 @@ This project uses Traefik as a reverse proxy and provides already a predefined s
 
 We can easily set up a project and don't bind the port of the project to the ports of our machine. Instead, we can connect a container to a prespecified Docker network and give it additional parameters using labels. These labels indicate details like the domain name of the service. An example project can be found in the folder `example`. Inside the file `example/docker-compose.yml` there are two things to keep in mind:
 
 - Use an external and a internal network. Only create the containers that should be accessed from the Traefik container to the `proxy` network and connect all other services together internally by assigning them to the `internal` network.
 
 - Add the labels to the container that should be accessed through the Traefik container and configure the host, port and backend name.
 
-After that, you have to modify the Traefik configuration with some details about the service:
-
-1. Edit the `/etc/hosts` file and point the domain of the service to localhost:
-
-    ```
-    127.0.0.1 traefik.aboutbits.local example.aboutbits.local
-    ```
-
-2. Paste the SSL certificate for the domain in the folder `certificates`.
-
-    Note: if you are searching for a solution to use certificates on your local development, check out the [certificate authority tools](https://github.com/aboutbits/certificate-authority-tools) that help you create your own certificate authorities. Usinge these tools you can generate and sign your own certificates that you then can use for development.
-
-3. Edit the file `traefik.toml` and add the new certificates that you copied in the previous step:
-
-    ```toml
-    [entryPoints.https.tls]
-      [[entryPoints.https.tls.certificates]]
-      keyFile = "/var/certificates/traefik.aboutbits.local.key"
-      certFile = "/var/certificates/traefik.aboutbits.local.pem"
-      [[entryPoints.https.tls.certificates]]
-      keyFile = "/var/certificates/service.aboutbits.local.key"
-      certFile = "/var/certificates/service.aboutbits.local.pem"
-    ```
-
-4. Reload the Traefik container by executing the following command:
-
-    ```bash
-    docker-compose up --detach --force-recreate
-    ```
-
-5. After these steps you can start the Docker containers of your service.
-
-## Running Traefik
-
-To start, reload or stop the Traefik container, simply execute one of the following command in your terminal:
-
-### Start Traefik
-
-```bash
-docker-compose up --detach
-```
-
-### Reload Traefik
-
-```bash
-docker-compose up --detach --force-recreate
-```
-
-### Stop Traefik
-
-```bash
-docker-compose down
-```
-
 ## Documentation
 
 Additional information about the usage of Traefik with Docker can be found [here](https://docs.traefik.io/configuration/backends/docker/). There you can find additional configuration settings and possible security concerns, that should be considered.
```

Comment:
This link is broken. The latest link to the documentation for docker is this one (https://doc.traefik.io/traefik/providers/docker/)